doc/languages-frameworks/python.section.md

```diff
@@ -1551,13 +1551,11 @@ In a `setup.py` or `setup.cfg` it is common to declare dependencies:
 
 ### Contributing guidelines
 
-Following rules are desired to be respected:
+The following rules are desired to be respected:
 
 * Python libraries are called from `python-packages.nix` and packaged with
   `buildPythonPackage`. The expression of a library should be in
   `pkgs/development/python-modules/<name>/default.nix`.
-* Libraries in `pkgs/top-level/python-packages.nix` are sorted
```

Comment:
Why is that line dropped? It is still true and almost all packages are sorted right now.

Reply:
It's a duplicate of the last bullet point.

Reply:
That's fair :)

Reply:
The last bullet point lacks the rationale, though.

Reply:
I added the rationale to the last point.